Library/Formula/libwebm.rb

```diff
@@ -0,0 +1,29 @@
+require 'formula'
```

Comment:
Not sure why when I run ```brew test libwebm``` it's failing with:

```
Error: libwebm: failed
Failed executing: ./test
/usr/local/Library/Homebrew/formula.rb:897:in `block in system'
/usr/local/Library/Homebrew/formula.rb:854:in `open'
/usr/local/Library/Homebrew/formula.rb:854:in `system'
/usr/local/Library/Formula/libwebm.rb:42:in `block in <class:Libwebm>'
/usr/local/Library/Homebrew/formula.rb:797:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:17:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:793:in `run_test'
/usr/local/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/test.rb:27:in `<main>'
```

But if I compile & run the code on my own it works.